### ISSUE TITLE
Ex-admin with config menu open is still able to change configs

### ISSUE THREAD

**Issue by ambravl** [parabola949/Werewolf#22]
If an user is admin of a group, and wwbot recognizes them as such, the user can open config menu, and isn't ever required to close it. If such user loses admin privileges (either by being kicked from group, exiting the group on their own or being removed as administrator by another admin), the bot won't recognize them as an admin anymore... But if the ex-admin still has the config menu open, they can keep making all sorts of changes in it until they close the menu (wwbot won't let them open a new menu, just won't close it when they stop being admin)

Basically wwbot will accept any change made from an open config menu even if it doesn't come from a current admin.

**Comment by parabola949** [comment]
On it right now, that's a big issue

**Comment by parabola949** [comment]
Resolved
